src/main/groovy/de/dkfz/roddy/execution/jobs/BEJob.groovy

```diff
@@ -140,23 +160,23 @@ class BEJob<J extends BEJob, JR extends BEJobResult> implements Comparable<BEJob
         return this
     }
 
-    BEJobResult getRunResult() {
+    JR getRunResult() {
```

Comment:
Here the generic should be used. Then also some highlighted type errors in IDEA vanish.